src/app/client/src/app/modules/shared-feature/components/popup/popup.component.html

```diff
@@ -0,0 +1,65 @@
+<!-- popup to edit -->
+
+<sui-modal [isClosable]="false" [size]="'normal'" #modal class="z-index-10000" *ngIf="formFieldProperties">
+  <i class="close icon" (click)="modal.deny()"></i>
+  <div class="content">
+    <div class="ui grid">
+      <div class="ten wide column mb-30">
+        <span class="ui header">{{formTitle}}</span>
+      </div>
+    </div>
+    <form class="ui form " id="EditProfileForm">
+      <div class="required field" *ngIf="board">
```

Comment:
This popup design is very specific to Diksha framework having board, medium, class and subject. This is ok for now though not ideal !